src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegend.module.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  margin: 10px 0 0 80px;
+  max-width: 950px;
+}
+
+.labelTextStrong {
+  font-size: 12px;
+  font-weight: var(--font-weight-bold);
+  margin-left: 10px;
+}
+
+.legendTable {
+  padding-top: 50px;
+}
+
+.legendTable table {
+  width: 100%;
+}
+.legendTable thead {
+  text-align: left;
+}
+.legendTable thead th,
+.legendTable tbody td {
+  padding-left: var(--table-cell-padding-left, 24px);
+  padding-right: var(--table-cell-padding-left, 32px);
+  padding-top: var(--table-cell-padding-top, 6px);
+  padding-bottom: var(--table-cell-padding-bottom, 6px);
+}
```

Comment:
I wonder why this was written this way. Accepting a variable from a parent and having a default value to fall back to makes sense for reusable components with configurable styles; but that's not what this component is. Could have been just a copypasta from the table component that I missed.

Wouldn't 

```
.legendTable thead th,
.legendTable tbody td {
  --table-cell-padding-left: 24px;
  --table-cell-padding-left: 32px;
  --table-cell-padding-top: 6px;
  --table-cell-padding-bottom: 6px;
}
```

make more sense here?

Reply:
@azangru Yes that would make more sense. I have pushed a commit with this change.